src/web/app/components/dialogs/MaterialTestGeneratorPanel/TableSettingForm.tsx

```diff
@@ -84,22 +140,45 @@ export default function TableSettingForm({
           value={key}
           onChange={(value) => handleSelectChange(value, index)}
         />
-        {['min', 'max'].map((prefix) => (
-          <UnitInput
-            key={`${prefix}-${key}`}
-            data-testid={`${prefix}-${key}`}
-            isInch={useInch}
-            className={styles.input}
-            value={detail[`${prefix}Value`]}
-            max={detail.max}
-            min={detail.min}
-            precision={useInch ? 4 : 0}
-            step={useInch ? 25.4 : 1}
-            // eslint-disable-next-line no-nested-ternary
-            addonAfter={key === 'strength' ? '%' : key === 'speed' ? lengthUnit : ''}
-            onChange={(value) => handleValueChange(key, prefix as 'min' | 'max', value)}
-          />
-        ))}
+        {['min', 'max'].map((prefix) => {
+          const addonAfter = () => {
+            switch (key) {
+              case 'strength':
+                return '%';
+              case 'speed':
+                return `${lengthUnit}/s`;
+              case 'fillInterval':
+                return lengthUnit;
```

Comment:
In 3a5739eede66f7cd85d0fc3677e1c5fa9279f8b6 fill interval has been changed to be unit-invariant.
Because inch users may not feel easier to set the value even if it uses inch